src/Sylius/Bundle/GridBundle/Twig/GridExtension.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\GridBundle\Twig;
+
+use Sylius\Bundle\GridBundle\Templating\Helper\GridHelper;
+use Sylius\Component\Grid\Definition\Action;
+use Sylius\Component\Grid\Definition\Field;
+use Sylius\Component\Grid\Definition\Filter;
+use Sylius\Component\Grid\View\GridView;
+
+/**
+ * @author Paweł Jędrzejewski <pawel@sylius.org>
+ */
+class GridExtension extends \Twig_Extension
+{
+    /**
+     * @var GridHelper
+     */
+    private $gridHelper;
+
+    /**
+     * @param GridHelper $gridHelper
+     */
+    public function __construct(GridHelper $gridHelper)
+    {
+        $this->gridHelper = $gridHelper;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getFunctions()
+    {
+        return [
+            new \Twig_SimpleFunction('sylius_grid_render', [$this, 'renderGrid'], ['is_safe' => ['html']]),
```

Comment:
If we replace the current callback `[$this, 'renderGrid']` with `[$this->gridHelper, 'renderGrid']`, we can get rid of these proxy methods in twig extension class.